lang/test/org/partiql/lang/eval/builtins/functions/MakeTimeEvaluationTest.kt

```diff
@@ -4,56 +4,47 @@ import org.junit.Test
-import org.partiql.lang.errors.Property
 import org.partiql.lang.eval.EvaluatorTestBase
 import org.partiql.lang.eval.builtins.Argument
 import org.partiql.lang.eval.builtins.ExprFunctionTestCase
 import org.partiql.lang.eval.builtins.checkInvalidArgType
 import org.partiql.lang.eval.builtins.checkInvalidArity
+import org.partiql.lang.eval.test.ExpectedResultFormat
 import org.partiql.lang.types.StaticType
 import org.partiql.lang.util.ArgumentsProviderBase
-import org.partiql.lang.util.to
+import org.partiql.lang.util.propertyValueMapOf
 
 class MakeTimeEvaluationTest : EvaluatorTestBase() {
     @ParameterizedTest
     @ArgumentsSource(MakeTimePassCases::class)
-    fun runPassTests(testCase: ExprFunctionTestCase) = assertEquals(eval(testCase.source).toString(), testCase.expected)
+    fun runPassTests(testCase: ExprFunctionTestCase) =
+        runEvaluatorTestCase(
+            query = testCase.source,
+            expectedResult = testCase.expectedLegacyModeResult,
+            expectedPermissiveModeResult = testCase.expectedPermissiveModeResult,
+            expectedResultFormat = ExpectedResultFormat.ION
+        )
 
     class MakeTimePassCases : ArgumentsProviderBase() {
         override fun getParameters(): List<Any> = listOf(
-            ExprFunctionTestCase("make_time(0, 0, 0.)", "00:00:00"),
-            ExprFunctionTestCase("make_time(0, 0, 0., 0)", "00:00:00+00:00"),
-            ExprFunctionTestCase("make_time(23, 12, 59.12345)", "23:12:59.12345"),
-            ExprFunctionTestCase("make_time(23, 12, 59.12345, 800)", "23:12:59.12345+13:20"),
-            ExprFunctionTestCase("make_time(23, 59, 59.999999999)", "23:59:59.999999999"),
-            ExprFunctionTestCase("make_time(23, 12, 59.12345, -800)", "23:12:59.12345-13:20"),
-            ExprFunctionTestCase("make_time(23, 59, 59.999999999, -1080)", "23:59:59.999999999-18:00"),
-            ExprFunctionTestCase("make_time(23, 59, 59.999999999, 1080)", "23:59:59.999999999+18:00"),
-            ExprFunctionTestCase("make_time(`23`, `12`, `59.12345`, `800`)", "23:12:59.12345+13:20"),
-            ExprFunctionTestCase("make_time(null, 59, 59.999999999)", "NULL"),
-            ExprFunctionTestCase("make_time(23, null, 59.999999999)", "NULL"),
-            ExprFunctionTestCase("make_time(23, 59, null)", "NULL"),
-            ExprFunctionTestCase("make_time(null, 59, 59.999999999, 1080)", "NULL"),
-            ExprFunctionTestCase("make_time(23, null, 59.999999999, 1080)", "NULL"),
-            ExprFunctionTestCase("make_time(23, 59, null, 1080)", "NULL"),
-            ExprFunctionTestCase("make_time(23, 59, 59.999999999, null)", "NULL"),
-            ExprFunctionTestCase("make_time(missing, 59,59.999999999, 1080)", "NULL"),
-            ExprFunctionTestCase("make_time(23, 59, missing, 1080)", "NULL"),
-            ExprFunctionTestCase("make_time(23, 59, 59.999999999, missing)", "NULL"),
-            ExprFunctionTestCase("make_time(23, 59, missing, null)", "NULL")
```

Comment:
Yes--they are redundant because null propagation is being implemented at the compiler level, not the function, and therefore these tests weren't even hitting the `make_time` `ExprFunction` implementation.  That functionality is being [tested elsewhere](https://github.com/partiql/partiql-lang-kotlin/blob/18c06a89b739bcb497795fe66cf6739c6efc270b/lang/test/org/partiql/lang/eval/EvaluatingCompilerUnknownValuesTest.kt#L29-L98).

Reply:
Even though null propagation is tested elsewhere, I think having some test ensuring that `make_time` is a null-propagating function could be valuable. Also there was a test on L32 that was deleted not related to null propagation.